packages/sdk/src/modules/moduleFromEndpoints/utils/getRequestSender.ts

```diff
@@ -1,13 +1,21 @@
 import {
   Options,
-  IncomingConfig,
+  RequestConfig,
   BaseConfig,
   ComputedConfig,
   HTTPClient,
   ErrorHandler,
+  RequestSender,
 } from "../types";
 
-export const getHTTPClient = (options: Options) => {
+/**
+ * Generates a `RequestSender` function configured according to the provided options.
+ *
+ * @remarks
+ * This function abstracts away the details of constructing request URLs, merging configurations,
```

Comment:
Can we just link to the `RequestSender` interface docs? I feel like you're repeating here what you've just written there

Reply:
imo this function might be used directly by the developers so it would be good to have it in here as well :)